app/services/report_actions/report_content.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+module ReportActions
+  class ReportContent
+    include Canaid::Helpers::PermissionsHelper
+
+    MY_MODULE_ADDONS_ELEMENTS = [].freeze
```

Comment:
If it's frozen how do you plan to extend it?

Reply:
Oh probably it is rubocop, I will fix it.